services/edus/pom.xml

```diff
@@ -47,6 +47,11 @@
       <groupId>org.apache.tomcat.embed</groupId>
       <artifactId>tomcat-embed-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-jcl</artifactId>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
Da ist keine Version definiert. Ist das Absicht?

Reply:
yes, it's managed by the spring or log4j imports